doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md

```diff
@@ -478,6 +495,35 @@ If an error is encountered during the `rollback` operation, an error is reported
 
 All the configuration update operations executed and the output displayed by the `rollback` command are stored in the systemd journal. They are also forwarded to the syslog. By storing the commands in the systemd-journal, the user will be able to search and display them easily at a later point in time. The `show rollback log` command reads the systemd-journal to display information about the `rollback` command that was previously executed or currently in progress.
 
+### 2.2.4 Replace
+The SONiC `replace` command can broadly classified into the following steps
+
+#### Stage-1 Get current ConfigDB JSON config
+The *ConfigDBConnector* class is used to obtain the running configuration in JSON format
+
+#### Stage-2 Get target config from the external user/system
+The external user/system to provide the full target ConfigDB config in JSON format.
+
+#### Stage-3 Validating the target config using YANG models
```

Comment:
This can be step 2 in replace case, because if the incoming config is not valid ,we do not even have to do current step 1 which is to get the current config DB.

Reply:
Good idea!

Reply:
updated locally

Reply:
updated